source/smeagle.cpp

```diff
@@ -21,6 +21,28 @@ using namespace smeagle;
 
 Smeagle::Smeagle(std::string _library) : library(std::move(_library)) {}
 
+// Determine if the library has exceptions with smeagle
+bool Smeagle::has_exceptions() {
+  Symtab *obj = NULL;
+  std::vector<ExceptionBlock *> exceptions;
+
+  // Read the library into the Symtab object, cut out early if there's error
+  if (not Symtab::openFile(obj, library)) {
+    throw std::runtime_error{"There was a problem reading from '" + library + "'"};
+  }
+
+  // Create a corpus
+  Corpus corpus(library);
```

Comment:
Do we need this? I don't see where it's being used.

Reply:
removed.